### PR TITLE
For example app make number of emoji columns dynamic based on screen width

### PR DESCRIPTION
**Pull request by timmaffett**
This change makes the number of columns for the emoji picker dynamic based on screen width.  When running the example on windows I was surpised to see this ridiculous result:

emoji_picker_flutter_example 4_30_2023 12_40_14 PM

Here instead is a more expected behavior for an emoji picker (this is the example with this PR added):

emoji_picker_flutter_example 4_30_2023 12_41_41 PM

For clarity I split out all of the emoji size related calculations to the top of the build method to better illustrate to the users what is going on.

For mobile versions the number of columns remains a reasonable number (for Pixel3 emulator it remains 7 as it was hard coded, for more modern phones the number of emoji columns increases as one would expect).

Thanks for the nice package!

### CODE REVIEW

**Comment by timmaffett** [comment]
I have moved the dynamic column sizing to it's own file.

File: example/lib/main-dynamic-columns.dart [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  @override
+  void dispose() {
+    _controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final screenSize = MediaQuery.of(context).size;
+    final emojiPadding = 9*2;
```
Comment (Fintasys) [inline example/lib/main-dynamic-columns.dart:28]: Any reason for not just writing `18` ? 🤔

**Comment by timmaffett** [comment]
I was just trying to illustrate it was a 9 pixel margin on both sides of
the emoji.
The compiler does the math so it is identical to writing 18.
Maybe I need to explain that in a comment or just put 18.. 😜

**Comment by Fintasys** [comment]
Comment makes more sense. After that I gonna merge, thanks for the work!

**Comment by timmaffett** [comment]
done and I ran `dart format` as well.

**Comment by Fintasys** [comment]
@timmaffett Sorry seems with my latest changes your PR still fails. Can you remove the `showRecentsTab` property from the config? 
```
error • The named parameter 'showRecentsTab' isn't defined • example/lib/main-dynamic-columns.dart:122:23 • undefined_named_parameter
```

**Pull request merged**